.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+<!--
+Welcome to the OpenTracing Go repo! 👋🎉
+
+- Please be respectful and considerate of others when commenting on issues.
+- Please search for existing issues in order to ensure we don't have duplicate bugs/feature requests.
+- Please don't ask questions here. If you have any question head to our gitter chat https://gitter.im/opentracing/public
+- Please provide as much information as possible so we all understand the issue.
+-->
+
+
+## Background
+Something that gives context about why this is an issue.
+
+## Problem
+Describe the problem. If the issue is about an improvement you can skip this. If possible, include a description of the impact/scope of the problem.
+
+## Proposal
+A proposal that from your POV would solve the problem or improve the existing situation. It should also include the impact.
+
+## Questions to address
+Questions that should be answered as outcome of this issue.
```

Comment:
Also not clear. Is this referring to open question the author might have even after offering a proposal?